app/controllers/users_controller.rb

```diff
@@ -49,8 +54,8 @@ def user_params
   end
 
   def phone_param
-    # We get the value from mobile_phone[full] on the form, intl-tel-input
-    # will put a normalised version of the number (with IDD code) there for us.
+    return params[:mobile_phone][:number] if params[:mobile_phone] && params[:mobile_phone][:number]
```

Comment:
I had a fight with haml and haml won, so the controller might get different parameters depending on where the (same) partial was linked from. Happy to fix if someone can provide the magic incantation to fix the way the partial is wrapped. Alternatively, we could not use the partial from the user/edit page, and have a different method to update only the phone number